src/Explorer.js

```diff
@@ -1796,6 +1796,20 @@ class Explorer extends React.PureComponent<Props, State> {
       };
 
       this.props.onEdit(print(newOperationDef));
+
+      const framesToWait = 5;
+      // This is inherently racey, but if it fails it's not a huge deal.
+      setTimeout(
```

Comment:
Can we do this in a way that's not racy?

You could hold some state in this function that keeps track of operations to scroll to. Then pass down an `onMount` function to the `RootView` component. The `RootView` can call `onMount` with a reference to itself--then the `onMount` function can scroll to the `RootView` ref into view and remove the operation from the list of operations we need to scroll to.